### ISSUE TITLE
Signing commits with GPG

### ISSUE THREAD

**Issue by seckin206** [github/VisualStudio#260]
I have followed the guide [here](https://jamesmckay.net/2016/02/signing-git-commits-with-gpg-on-windows/) to be able to sign my commits.

I tested in cmd and my commits are signed as expected. However, I cannot sign my commits using visual studio. Is the extension ignoring my global git config?

**Comment by shana** [comment]
@seckin206 This repo is for the GitHub extension, which doesn't handle the git support in Visual Studio -  that's owned by Microsoft. You should report this via the feedback tool in Visual Studio.

/cc @acangialosi @clboles

**Comment by seckin206** [comment]
Thanks for the explanation, will do.

**Comment by cbcoutinho** [comment]
@seckin206 I am trying to find more info on signing commits in visual studio team services - did you find anything more on this?

**Comment by seckin206** [comment]
@cbcoutinho Unfortunately, I couldn't find much info at that time. I don't have the use case anymore since I switch to using another git client.

**Comment by cbcoutinho** [comment]
@seckin206 Thanks for the reply, I'll assume the situation hasn't changed.

**Comment by weitzhandler** [comment]
For the reference: https://developercommunity.visualstudio.com/idea/531972/let-the-git-plugin-to-sign-commits-with-gpg-key-wi.html

Related: #513.

**Comment by ArmaanMcleod** [comment]
TLDR

Do this:

```bash
git config --global commit.gpgsign true
git config --global gpg.program "C:/Program Files/Git/usr/bin/gpg.exe" # Or wherever your gpg executable is 
```
